### PR TITLE
Elixir plugin

### PR DESCRIPTION
**Pull request by deadtrickster**
Elixir files can be placed to `src` or `lib`. 
Also this part of Elixir quickstart might be useful: http://elixir-lang.org/crash-course.html#adding-elixir-to-existing-erlang-programs

### CODE REVIEW

File: build.config [hunk 1]
```diff
@@ -22,3 +22,4 @@ plugins/eunit
 plugins/relx
 plugins/shell
 plugins/triq
+plugins/elixir
```
Comment (essen) [inline build.config]: Alphabetical order please.

**Comment by essen** [comment]
No feedback, so I'm OK to merge after the change is done and I test.

**Comment by essen** [comment]
Might be useful to add an Elixir file template and to write a test.

**Comment by deadtrickster** [comment]
where to add template? also tests are failing:

```
Setting up app.
make[1]: Entering directory '/home/___/Projects/erlang.mk/test/app1'
make[1]: Leaving directory '/home/___/Projects/erlang.mk/test/app1'
app: Testing the 'app' target.
Checking the modules line in the generated .app file.
make[1]: Entering directory '/home/___/Projects/erlang.mk/test/app1'
grep: app1/ebin/app1.app: No such file or directory
/bin/sh: 1: [: 0: unexpected operator
Makefile:47: recipe for target 'app' failed
make: *** [app] Error 2
```

**Comment by essen** [comment]
I will check the test. For the template, you can add it in plugins/bootstrap.mk, should be straightforward enough.

**Comment by essen** [comment]
OK you messed up somewhere, don't know what you did, but we can look tomorrow if you want. :-)

**Pull request closed without merging**